worldgen.h

```diff
@@ -41,6 +41,7 @@ typedef struct terrain_map
     char grid[GRID_HEIGHT][GRID_WIDTH];
     Location_t location;
     Gates_t gates;
+    int visited, cost, distance;
```

Comment:
struct definition should always have each parameter on a separate line I dont care they share the same type